src/components/ErrorPages/SessionExpired.tsx

```diff
@@ -12,27 +110,128 @@ export default function SessionExpired() {
     toast.dismiss();
   }, []);
 
+  const { seconds, breathState, timeStr, scaleFactor, shouldShowTime } =
+    useSessionTimer();
+
+  const renderSegmentedDigit = (digit: string) => {
+    const map = digitMaps[digit] || digitMaps["0"];
+
+    return (
+      <div className="inline-block mx-0.5">
+        {map.map((row, rowIndex) => (
+          <div key={rowIndex} className="flex">
+            {row.map((cell, cellIndex) => (
+              <div
+                key={cellIndex}
+                className={`w-1.5 h-1.5 m-px ${
+                  cell ? "bg-gray-400" : "bg-transparent"
+                }`}
+              ></div>
+            ))}
+          </div>
+        ))}
+      </div>
+    );
+  };
+
+  const renderSegmentedTime = (timeStr: string) => (
+    <div
+      className={cn("flex items-center justify-center transform origin-center")}
+      style={{ transform: `scale(${scaleFactor})` }}
+    >
+      {timeStr.split("").map((char, index) => (
+        <div key={index}>{renderSegmentedDigit(char)}</div>
+      ))}
+    </div>
+  );
+
   return (
-    <div className="flex h-screen items-center justify-center text-center">
-      <div className="w-[500px] text-center">
-        <img
-          src="/images/session_expired.svg"
-          alt={t("session_expired")}
-          className="mx-auto w-1/2"
-        />
-        <h1>{t("session_expired")}</h1>
-        <p>
-          {t("session_expired_msg")}
-          <br />
-          <br />
+    <div
+      className={cn(
+        "flex flex-col items-center justify-center w-full h-screen bg-white",
+      )}
+    >
+      <div className="relative flex items-center justify-center w-72 h-72 md:w-96 md:h-96">
+        {/* Ripples */}
+        <div
+          className={cn("absolute inset-0 flex items-center justify-center")}
+        >
           <div
-            onClick={signOut}
-            className="mt-5 inline-block cursor-pointer rounded-lg bg-primary-600 px-4 py-2 text-white hover:bg-primary-400"
+            className={cn(
+              "absolute w-80 h-80 bg-emerald-400/20 rounded-full animate-ping-slow",
+            )}
+          ></div>
+          <div
+            className={cn(
+              "absolute w-64 h-64 bg-emerald-500/20 rounded-full animate-ping-medium",
+            )}
+          ></div>
+          <div
+            className={cn(
+              "absolute w-48 h-48 bg-emerald-600/20 rounded-full animate-ping-fast",
```

Comment:
modify `h-* w-*` everywhere to be `size-*`

```suggestion
              "absolute size-48 bg-emerald-600/20 rounded-full animate-ping-fast",
```

Reply:
modified height and weight everywhere